src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -48,20 +45,35 @@
  * </p>
  * <p>
  * Package deprecation is a exception to the rule of always using the
- * javadoc tag and annotation to deprecate.  Only the package-info.java
- * file can contain a Deprecated annotation and it CANNOT contain
- * a deprecated javadoc tag.  This is the case with
- * Sun's javadoc tool released with JDK 1.6.0_11.  As a result, this check
- * does not deal with Deprecated packages in any way.  <b>No official
- * documentation was found confirming this behavior is correct
- * (of the javadoc tool).</b>
+ * javadoc tag and annotation to deprecate.  It is not clear if the javadoc
+ * tool will support it or not as newer versions keep flip flopping on if
+ * it is supported or will cause an error.  The deprecated javadoc tag is
```

Comment:
please put link to issue against javadoc tool.

Reply:
Done.